src/lib/index.ts

```diff
@@ -45,8 +45,12 @@ class AssetEntry {
         }
     }
     private applyScript(tag: HTMLScriptElement) {
+        const start = /<script>/gi;
+        const end = /<\/script>gi;
         tag.removeAttribute('src');
-        tag.innerHTML = this.content.toString();
+        var js = this.content.toString();
+        js = content.replace(start,' ').replace(end,' ');
```

Comment:
Does it have to be `' '` than `''`?

Reply:
Not really - my understanding was this is in the content of the js that gets embedded - i don't think it matters.

Reply:
Pushed '' in last fix